fortimanager_consts.py

```diff
@@ -0,0 +1,14 @@
+
+# Login information
+ACTION_PATH = '/jsonrpc'
+LOGIN_URL = '/sys/login/user'
+TEST_CONNECTIVITY_URL = '/sys/status'
+
+LOGIN_ERROR_MSG = 'login failed'
+ERROR_MSG_UNAVAILABLE = 'Error message unavailable'
+
+# Firewall Endpoints
+ADOM_FIREWALL_ENDPOINT = '/pm/config/adom/{adom}/pkg/{pkg}/firewall/policy'
+GLOBAL_FIREWALL_ENDPOINT = '/pm/config/global/pkg/{pkg}/global/{policy_type}/policy'
```

Comment:
should we remove global endpoints from the consts file since we aren't using them in this release?

Reply:
I will comment it out